src/Job/DeleteStaticCacheJob.php

```diff
@@ -6,31 +6,17 @@
 use SilverStripe\StaticPublishQueue\Publisher;
 
 /**
- * Class DeleteStaticCacheJob
- * remove pages from static cache based on list of URLs
- *
- * @package SilverStripe\StaticPublishQueue\Job
+ * Remove pages from static cache based on list of URLs
  */
 class DeleteStaticCacheJob extends Job
 {
-    /**
-     * @var int
-     * @config
-     */
-    private static $chunk_size = 2000;
-
-    /**
-     * @return string
-     */
+    private static int $chunk_size = 2000;
```

Comment:
I know this seems like it's splitting hairs though adding a type to a private static is technically an API change 
```suggestion
    private static $chunk_size = 2000;
```